client/cs_scheduler.cpp

```diff
@@ -676,9 +676,10 @@ int CLIENT_STATE::handle_scheduler_reply(
     }
 
     // if the scheduler reply includes global preferences,
+    // and we doesn't join any account manager,
```

Comment:
small but important change needed: "and we currently don't use an account manager"